contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,166 @@
+    message_content = message_content.split('/')
+
+    try:
+        # this will work if the information was entered correctly
+        user = message_content[1]
+        repo_owner = message_content[2]
+        repo = message_content[3]
+        issue = message_content[4]
+        comment_body = message_content[5]
+
+        return {'user': user, 'repo_owner': repo_owner, 'repo': repo, 'issue': issue, 'comment_body': comment_body}
+    except IndexError:
+        # if the information was not entered correctly, then this will catch the error
+        # sets value of index_error_happened to true, to avoid getting duplicate error messages.
+        global index_error_happened
+        index_error_happened = True
+        message = "It doesn't look like the information was entered in the correct format." \
+                  " Did you input it like this? " \
+                  "'/<username>/<repository_owner>/<repository>/<issue_number>/<your_comment>'."
+        send_message(client, message, original_sender)
+        print('there was an error with the information you entered')
+
+
+def handle_input(client, original_content, original_sender):
+
+    try:
+
+        params = get_values_message(original_content, client, original_sender)
+
+        status_code = send_to_github(params['user'], params['repo_owner'], params['repo'],
+                                     params['issue'], params['comment_body'])
+
+        if status_code == 201:
+            # sending info to github was successful!
+            reply_message = "You commented on issue number " + params['issue'] + " under " + \
+                            params['repo_owner'] + "'s repository " + params['repo'] + "!"
+
+            send_message(client, reply_message, original_sender)
+
+        else:
+            # sending info to github did not work
+            reply_message = ":( There was a problem commenting on issue number " + params['issue'] + " under " + \
+                            params['repo_owner'] + "'s repository " + params['repo'] + \
+                            ". Did you enter the information in the correct format?"
+
+            send_message(client, reply_message, original_sender)
+
+    except TypeError:
+        if not index_error_happened:
+            message = "It doesn't look like the information was entered in the correct format." \
+                      " Did you input it like this? " \
+                      "'/<username>/<repository_owner>/<repository>/<issue_number>/<your_comment>'."
+            send_message(client, message, original_sender)
+            print('there was an error with the information you entered')
+
+
+def get_github_token():
+    # get github OAuth token from file where stored
+    if os.path.exists('~/path_github_token.txt'):
+        with cd('~/'):
+            path_to_token = open('~/path_to_token').read().strip()
+        with cd(path_to_token):
+            oauth_token = open('github_token.txt').read().strip()
```

Comment:
This is a relative path right? So you are looking for the file in the current directory.